### ISSUE TITLE
二分类怎么做？

### ISSUE THREAD

**Issue by ahuxjz** [yassouali/pytorch_segmentation#11]

**Comment by yassouali** [comment]
Hi, can you provide more information about your issue ?

**Comment by ahuxjz** [comment]
> Hi, can you provide more information about your issue ?
I have two categories, including background, background pixel value (0,0,0) and target pixel value (255,255,255). How can I modify the code?  What are the specific changes?  Thank you for your answer!

**Comment by ahuxjz** [comment]
> > 嗨，您能提供有关您的问题的更多信息吗？
> > 我有两个类别，包括背景，背景像素值（0,0,0）和目标像素值（255,255,255）。我该如何修改代码？有哪些具体变化？谢谢您的回答！

Because one of my class pixels is (255, 255, 255), is it feasible to set my ignore_label to 255 and palette to a picture?  Are there any other details?[
![TIM图片20190912110247](https://user-images.githubusercontent.com/44185528/64750894-0be0f100-d54d-11e9-9cde-4dd8c5e6b3ea.png)
](url)

**Comment by ahuxjz** [comment]
> 只需将加载器中的类数设置为2即可设置，您将拥有两个标签，一个和零。

Thank you for your answer. I trained once yesterday, 5 epochs, no data enhancements. The predictions are all black images. Is my code error or poor performance?

**Comment by yassouali** [comment]
Yeah, I think there is some problem with your labels, see if the masks in tensorboard are correct.

**Comment by GewelsJI** [comment]
> > 只需将加载器中的类数设置为2即可设置，您将拥有两个标签，一个和零。
> 
> Thank you for your answer. I trained once yesterday, 5 epochs, no data enhancements. The predictions are all black images. Is my code error or poor performance?

Could you share your code what you edit? I face the same problem now

Thanks
